[BUG] - <title> 插件编译时报错

### Description

[2026-01-29 16:53:49] Project community-serve is initializing
[2026-01-29 16:53:49] SourcesProcessJava Project:community-serve:main APTOptions:{}
[2026-01-29 16:53:51] java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:118)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$sourcesProcessJava$2.invokeSuspend$lambda$1$0(JimmerBuddy.kt:393)
	at com.intellij.openapi.application.ReadAction.lambda$run$1(ReadAction.java:54)
	at com.intellij.openapi.application.impl.AppImplKt$rethrowCheckedExceptions$2.invoke(appImpl.kt:126)
	at com.intellij.platform.locking.impl.NestedLocksThreadingSupport.runReadAction(NestedLocksThreadingSupport.kt:856)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:1069)
	at com.intellij.openapi.application.ReadAction.compute(ReadAction.java:66)
	at com.intellij.openapi.application.ReadAction.run(ReadAction.java:53)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$sourcesProcessJava$2.invokeSuspend(JimmerBuddy.kt:354)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$sourcesProcessJava$2.invoke(JimmerBuddy.kt)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$sourcesProcessJava$2.invoke(JimmerBuddy.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:44)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:166)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.platform.util.progress.ProgressPipeImpl.collectProgressUpdates(ProgressPipe.kt:43)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2$2.invokeSuspend(PlatformTaskSupport.kt:120)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2$2.invoke(PlatformTaskSupport.kt)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2$2.invoke(PlatformTaskSupport.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:44)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:157)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2.invokeSuspend(PlatformTaskSupport.kt:119)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2.invoke(PlatformTaskSupport.kt)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2.invoke(PlatformTaskSupport.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:44)
	at kotlinx.coroutines.CoroutineScopeKt.coroutineScope(CoroutineScope.kt:285)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport.withBackgroundProgressInternal(PlatformTaskSupport.kt:103)
	at com.intellij.platform.ide.progress.TasksKt.withBackgroundProgress(tasks.kt:83)
	at com.intellij.platform.ide.progress.TasksKt.withBackgroundProgress(tasks.kt:23)
	at com.intellij.openapi.progress.TasksKt.withBackgroundProgress(tasks.kt:34)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace.sourcesProcessJava(JimmerBuddy.kt:336)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$init$1.invokeSuspend(JimmerBuddy.kt:212)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:610)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runDefaultDispatcherTask(CoroutineScheduler.kt:1194)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:906)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:775)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:762)
Caused by: java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:118)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at cn.enaium.jimmer.buddy.utility.AptKt$createAnnotationMirror$1.getElementValues$lambda$4$0(apt.kt:930)
	at cn.enaium.jimmer.buddy.utility.AptKt$createAnnotationValue$2.getValue(apt.kt:863)
	at cn.enaium.jimmer.buddy.utility.AptKt$createAnnotationValue$2.toString(apt.kt:884)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableProp.<init>(ImmutableProp.java:302)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableType.<init>(ImmutableType.java:277)
	at org.babyfish.jimmer.apt.Context.getImmutableType(Context.java:273)
	at org.babyfish.jimmer.apt.immutable.ImmutableProcessor.parseImmutableTypes(ImmutableProcessor.java:65)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	... 39 more
Caused by: java.lang.IllegalAccessException: class cn.enaium.jimmer.buddy.utility.AptKt cannot access a member of class cn.enaium.jimmer.buddy.Utility$21_Proxy with modifiers "public"
	at java.base/jdk.internal.reflect.Reflection.newIllegalAccessException(Reflection.java:394)
	at java.base/java.lang.reflect.AccessibleObject.checkAccess(AccessibleObject.java:714)
	at java.base/java.lang.reflect.Method.invoke(Method.java:571)
	at cn.enaium.jimmer.buddy.utility.AptKt.findAnnotation$lambda$0$2(apt.kt:747)
	at cn.enaium.jimmer.buddy.Utility$21_Proxy.value(Utility.java)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	... 48 more

[2026-01-29 16:53:51] SourcesProcessJava Project:community-serve:test APTOptions:{}
[2026-01-29 16:53:52] java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:118)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$sourcesProcessJava$2.invokeSuspend$lambda$1$0(JimmerBuddy.kt:393)
	at com.intellij.openapi.application.ReadAction.lambda$run$1(ReadAction.java:54)
	at com.intellij.openapi.application.impl.AppImplKt$rethrowCheckedExceptions$2.invoke(appImpl.kt:126)
	at com.intellij.platform.locking.impl.NestedLocksThreadingSupport.runReadAction(NestedLocksThreadingSupport.kt:856)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:1069)
	at com.intellij.openapi.application.ReadAction.compute(ReadAction.java:66)
	at com.intellij.openapi.application.ReadAction.run(ReadAction.java:53)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$sourcesProcessJava$2.invokeSuspend(JimmerBuddy.kt:354)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$sourcesProcessJava$2.invoke(JimmerBuddy.kt)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$sourcesProcessJava$2.invoke(JimmerBuddy.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:44)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:166)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.platform.util.progress.ProgressPipeImpl.collectProgressUpdates(ProgressPipe.kt:43)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2$2.invokeSuspend(PlatformTaskSupport.kt:120)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2$2.invoke(PlatformTaskSupport.kt)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2$2.invoke(PlatformTaskSupport.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:44)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:157)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2.invokeSuspend(PlatformTaskSupport.kt:119)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2.invoke(PlatformTaskSupport.kt)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2.invoke(PlatformTaskSupport.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:44)
	at kotlinx.coroutines.CoroutineScopeKt.coroutineScope(CoroutineScope.kt:285)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport.withBackgroundProgressInternal(PlatformTaskSupport.kt:103)
	at com.intellij.platform.ide.progress.TasksKt.withBackgroundProgress(tasks.kt:83)
	at com.intellij.platform.ide.progress.TasksKt.withBackgroundProgress(tasks.kt:23)
	at com.intellij.openapi.progress.TasksKt.withBackgroundProgress(tasks.kt:34)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace.sourcesProcessJava(JimmerBuddy.kt:336)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$init$1.invokeSuspend(JimmerBuddy.kt:212)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:610)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runDefaultDispatcherTask(CoroutineScheduler.kt:1194)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:906)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:775)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:762)
Caused by: java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:118)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at cn.enaium.jimmer.buddy.utility.AptKt$createAnnotationMirror$1.getElementValues$lambda$4$0(apt.kt:930)
	at cn.enaium.jimmer.buddy.utility.AptKt$createAnnotationValue$2.getValue(apt.kt:863)
	at cn.enaium.jimmer.buddy.utility.AptKt$createAnnotationValue$2.toString(apt.kt:884)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableProp.<init>(ImmutableProp.java:302)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableType.<init>(ImmutableType.java:277)
	at org.babyfish.jimmer.apt.Context.getImmutableType(Context.java:273)
	at org.babyfish.jimmer.apt.immutable.ImmutableProcessor.parseImmutableTypes(ImmutableProcessor.java:65)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	... 39 more
Caused by: java.lang.IllegalAccessException: class cn.enaium.jimmer.buddy.utility.AptKt cannot access a member of class cn.enaium.jimmer.buddy.Utility$21_Proxy with modifiers "public"
	at java.base/jdk.internal.reflect.Reflection.newIllegalAccessException(Reflection.java:394)
	at java.base/java.lang.reflect.AccessibleObject.checkAccess(AccessibleObject.java:714)
	at java.base/java.lang.reflect.Method.invoke(Method.java:571)
	at cn.enaium.jimmer.buddy.utility.AptKt.findAnnotation$lambda$0$2(apt.kt:747)
	at cn.enaium.jimmer.buddy.Utility$21_Proxy.value(Utility.java)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	... 48 more

[2026-01-29 16:53:52] DtoProcessJava Project:community-serve:main APTOptions:{}
[2026-01-29 16:53:53] java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:118)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at cn.enaium.jimmer.buddy.utility.AptKt$createAnnotationMirror$1.getElementValues$lambda$4$0(apt.kt:930)
	at cn.enaium.jimmer.buddy.utility.AptKt$createAnnotationValue$2.getValue(apt.kt:863)
	at cn.enaium.jimmer.buddy.utility.AptKt$createAnnotationValue$2.toString(apt.kt:884)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableProp.<init>(ImmutableProp.java:302)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableType.<init>(ImmutableType.java:277)
	at org.babyfish.jimmer.apt.Context.getImmutableType(Context.java:273)
	at org.babyfish.jimmer.apt.Context.getImmutableType(Context.java:282)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableProp.getTargetType(ImmutableProp.java:781)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableProp.getIdViewBaseProp(ImmutableProp.java:922)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableProp.getIdViewBaseProp(ImmutableProp.java:29)
	at org.babyfish.jimmer.dto.compiler.DtoTypeBuilder.isAutoScalar(DtoTypeBuilder.java:515)
	at org.babyfish.jimmer.dto.compiler.DtoTypeBuilder.handleMacro(DtoTypeBuilder.java:183)
	at org.babyfish.jimmer.dto.compiler.DtoTypeBuilder.<init>(DtoTypeBuilder.java:131)
	at org.babyfish.jimmer.dto.compiler.CompilerContext.add(CompilerContext.java:150)
	at org.babyfish.jimmer.dto.compiler.DtoCompiler.compile(DtoCompiler.java:89)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invokeSuspend$lambda$0(JimmerBuddy.kt:452)
	at com.intellij.openapi.application.ReadAction.lambda$run$1(ReadAction.java:54)
	at com.intellij.openapi.application.impl.AppImplKt$rethrowCheckedExceptions$2.invoke(appImpl.kt:126)
	at com.intellij.platform.locking.impl.NestedLocksThreadingSupport.runReadAction(NestedLocksThreadingSupport.kt:856)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:1069)
	at com.intellij.openapi.application.ReadAction.compute(ReadAction.java:66)
	at com.intellij.openapi.application.ReadAction.run(ReadAction.java:53)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invokeSuspend(JimmerBuddy.kt:429)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invoke(JimmerBuddy.kt)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invoke(JimmerBuddy.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:44)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:166)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.platform.util.progress.ProgressPipeImpl.collectProgressUpdates(ProgressPipe.kt:43)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2$2.invokeSuspend(PlatformTaskSupport.kt:120)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2$2.invoke(PlatformTaskSupport.kt)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2$2.invoke(PlatformTaskSupport.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:44)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:157)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2.invokeSuspend(PlatformTaskSupport.kt:119)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2.invoke(PlatformTaskSupport.kt)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2.invoke(PlatformTaskSupport.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:44)
	at kotlinx.coroutines.CoroutineScopeKt.coroutineScope(CoroutineScope.kt:285)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport.withBackgroundProgressInternal(PlatformTaskSupport.kt:103)
	at com.intellij.platform.ide.progress.TasksKt.withBackgroundProgress(tasks.kt:83)
	at com.intellij.platform.ide.progress.TasksKt.withBackgroundProgress(tasks.kt:23)
	at com.intellij.openapi.progress.TasksKt.withBackgroundProgress(tasks.kt:34)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace.dtoProcessJava(JimmerBuddy.kt:426)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$init$1.invokeSuspend(JimmerBuddy.kt:213)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:610)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runDefaultDispatcherTask(CoroutineScheduler.kt:1194)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:906)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:775)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:762)
Caused by: java.lang.IllegalAccessException: class cn.enaium.jimmer.buddy.utility.AptKt cannot access a member of class cn.enaium.jimmer.buddy.Utility$21_Proxy with modifiers "public"
	at java.base/jdk.internal.reflect.Reflection.newIllegalAccessException(Reflection.java:394)
	at java.base/java.lang.reflect.AccessibleObject.checkAccess(AccessibleObject.java:714)
	at java.base/java.lang.reflect.Method.invoke(Method.java:571)
	at cn.enaium.jimmer.buddy.utility.AptKt.findAnnotation$lambda$0$2(apt.kt:747)
	at cn.enaium.jimmer.buddy.Utility$21_Proxy.value(Utility.java)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	... 54 more

[2026-01-29 16:53:53] DtoProcessJava Project:community-serve:main APTOptions:{}
[2026-01-29 16:53:53] DtoProcessJava Project:community-serve:main APTOptions:{}
[2026-01-29 16:53:54] DtoProcessJava Project:community-serve:main APTOptions:{}
[2026-01-29 16:53:54] DtoProcessJava Project:community-serve:main APTOptions:{}
[2026-01-29 16:53:54] DtoProcessJava Project:community-serve:main APTOptions:{}
[2026-01-29 16:53:55] java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:118)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at cn.enaium.jimmer.buddy.utility.AptKt$createAnnotationMirror$1.getElementValues$lambda$4$0(apt.kt:930)
	at cn.enaium.jimmer.buddy.utility.AptKt$createAnnotationValue$2.getValue(apt.kt:863)
	at cn.enaium.jimmer.buddy.utility.AptKt$createAnnotationValue$2.toString(apt.kt:884)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableProp.<init>(ImmutableProp.java:302)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableType.<init>(ImmutableType.java:277)
	at org.babyfish.jimmer.apt.Context.getImmutableType(Context.java:273)
	at org.babyfish.jimmer.apt.Context.getImmutableType(Context.java:282)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableProp.getTargetType(ImmutableProp.java:781)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableProp.getIdViewBaseProp(ImmutableProp.java:922)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableProp.getIdViewBaseProp(ImmutableProp.java:29)
	at org.babyfish.jimmer.dto.compiler.DtoTypeBuilder.isAutoScalar(DtoTypeBuilder.java:515)
	at org.babyfish.jimmer.dto.compiler.DtoTypeBuilder.handleMacro(DtoTypeBuilder.java:260)
	at org.babyfish.jimmer.dto.compiler.DtoTypeBuilder.<init>(DtoTypeBuilder.java:131)
	at org.babyfish.jimmer.dto.compiler.CompilerContext.add(CompilerContext.java:150)
	at org.babyfish.jimmer.dto.compiler.DtoCompiler.compile(DtoCompiler.java:89)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invokeSuspend$lambda$0(JimmerBuddy.kt:452)
	at com.intellij.openapi.application.ReadAction.lambda$run$1(ReadAction.java:54)
	at com.intellij.openapi.application.impl.AppImplKt$rethrowCheckedExceptions$2.invoke(appImpl.kt:126)
	at com.intellij.platform.locking.impl.NestedLocksThreadingSupport.runReadAction(NestedLocksThreadingSupport.kt:856)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:1069)
	at com.intellij.openapi.application.ReadAction.compute(ReadAction.java:66)
	at com.intellij.openapi.application.ReadAction.run(ReadAction.java:53)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invokeSuspend(JimmerBuddy.kt:429)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invoke(JimmerBuddy.kt)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invoke(JimmerBuddy.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:44)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:166)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.platform.util.progress.ProgressPipeImpl.collectProgressUpdates(ProgressPipe.kt:43)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2$2.invokeSuspend(PlatformTaskSupport.kt:120)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2$2.invoke(PlatformTaskSupport.kt)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2$2.invoke(PlatformTaskSupport.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:44)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:157)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2.invokeSuspend(PlatformTaskSupport.kt:119)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2.invoke(PlatformTaskSupport.kt)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2.invoke(PlatformTaskSupport.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:44)
	at kotlinx.coroutines.CoroutineScopeKt.coroutineScope(CoroutineScope.kt:285)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport.withBackgroundProgressInternal(PlatformTaskSupport.kt:103)
	at com.intellij.platform.ide.progress.TasksKt.withBackgroundProgress(tasks.kt:83)
	at com.intellij.platform.ide.progress.TasksKt.withBackgroundProgress(tasks.kt:23)
	at com.intellij.openapi.progress.TasksKt.withBackgroundProgress(tasks.kt:34)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace.dtoProcessJava(JimmerBuddy.kt:426)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$init$1.invokeSuspend(JimmerBuddy.kt:213)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:610)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runDefaultDispatcherTask(CoroutineScheduler.kt:1194)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:906)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:775)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:762)
Caused by: java.lang.IllegalAccessException: class cn.enaium.jimmer.buddy.utility.AptKt cannot access a member of class cn.enaium.jimmer.buddy.Utility$21_Proxy with modifiers "public"
	at java.base/jdk.internal.reflect.Reflection.newIllegalAccessException(Reflection.java:394)
	at java.base/java.lang.reflect.AccessibleObject.checkAccess(AccessibleObject.java:714)
	at java.base/java.lang.reflect.Method.invoke(Method.java:571)
	at cn.enaium.jimmer.buddy.utility.AptKt.findAnnotation$lambda$0$2(apt.kt:747)
	at cn.enaium.jimmer.buddy.Utility$21_Proxy.value(Utility.java)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	... 54 more

[2026-01-29 16:53:55] DtoProcessJava Project:community-serve:main APTOptions:{}
[2026-01-29 16:53:55] java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:118)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at cn.enaium.jimmer.buddy.utility.AptKt$createAnnotationMirror$1.getElementValues$lambda$4$0(apt.kt:930)
	at cn.enaium.jimmer.buddy.utility.AptKt$createAnnotationValue$2.getValue(apt.kt:863)
	at cn.enaium.jimmer.buddy.utility.AptKt$createAnnotationValue$2.toString(apt.kt:884)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableProp.<init>(ImmutableProp.java:302)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableType.<init>(ImmutableType.java:277)
	at org.babyfish.jimmer.apt.Context.getImmutableType(Context.java:273)
	at org.babyfish.jimmer.apt.Context.getImmutableType(Context.java:282)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableProp.getTargetType(ImmutableProp.java:781)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableProp.getIdViewBaseProp(ImmutableProp.java:922)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableProp.getIdViewBaseProp(ImmutableProp.java:29)
	at org.babyfish.jimmer.dto.compiler.DtoTypeBuilder.isAutoScalar(DtoTypeBuilder.java:515)
	at org.babyfish.jimmer.dto.compiler.DtoTypeBuilder.handleMacro(DtoTypeBuilder.java:183)
	at org.babyfish.jimmer.dto.compiler.DtoTypeBuilder.<init>(DtoTypeBuilder.java:131)
	at org.babyfish.jimmer.dto.compiler.CompilerContext.add(CompilerContext.java:150)
	at org.babyfish.jimmer.dto.compiler.DtoCompiler.compile(DtoCompiler.java:89)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invokeSuspend$lambda$0(JimmerBuddy.kt:452)
	at com.intellij.openapi.application.ReadAction.lambda$run$1(ReadAction.java:54)
	at com.intellij.openapi.application.impl.AppImplKt$rethrowCheckedExceptions$2.invoke(appImpl.kt:126)
	at com.intellij.platform.locking.impl.NestedLocksThreadingSupport.runReadAction(NestedLocksThreadingSupport.kt:856)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:1069)
	at com.intellij.openapi.application.ReadAction.compute(ReadAction.java:66)
	at com.intellij.openapi.application.ReadAction.run(ReadAction.java:53)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invokeSuspend(JimmerBuddy.kt:429)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invoke(JimmerBuddy.kt)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invoke(JimmerBuddy.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:44)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:166)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.platform.util.progress.ProgressPipeImpl.collectProgressUpdates(ProgressPipe.kt:43)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2$2.invokeSuspend(PlatformTaskSupport.kt:120)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2$2.invoke(PlatformTaskSupport.kt)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2$2.invoke(PlatformTaskSupport.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:44)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:157)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2.invokeSuspend(PlatformTaskSupport.kt:119)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2.invoke(PlatformTaskSupport.kt)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2.invoke(PlatformTaskSupport.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:44)
	at kotlinx.coroutines.CoroutineScopeKt.coroutineScope(CoroutineScope.kt:285)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport.withBackgroundProgressInternal(PlatformTaskSupport.kt:103)
	at com.intellij.platform.ide.progress.TasksKt.withBackgroundProgress(tasks.kt:83)
	at com.intellij.platform.ide.progress.TasksKt.withBackgroundProgress(tasks.kt:23)
	at com.intellij.openapi.progress.TasksKt.withBackgroundProgress(tasks.kt:34)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace.dtoProcessJava(JimmerBuddy.kt:426)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$init$1.invokeSuspend(JimmerBuddy.kt:213)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:610)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runDefaultDispatcherTask(CoroutineScheduler.kt:1194)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:906)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:775)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:762)
Caused by: java.lang.IllegalAccessException: class cn.enaium.jimmer.buddy.utility.AptKt cannot access a member of class cn.enaium.jimmer.buddy.Utility$21_Proxy with modifiers "public"
	at java.base/jdk.internal.reflect.Reflection.newIllegalAccessException(Reflection.java:394)
	at java.base/java.lang.reflect.AccessibleObject.checkAccess(AccessibleObject.java:714)
	at java.base/java.lang.reflect.Method.invoke(Method.java:571)
	at cn.enaium.jimmer.buddy.utility.AptKt.findAnnotation$lambda$0$2(apt.kt:747)
	at cn.enaium.jimmer.buddy.Utility$21_Proxy.value(Utility.java)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	... 54 more

[2026-01-29 16:53:55] DtoProcessJava Project:community-serve:main APTOptions:{}
[2026-01-29 16:53:56] java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:118)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at cn.enaium.jimmer.buddy.utility.AptKt$createAnnotationMirror$1.getElementValues$lambda$4$0(apt.kt:930)
	at cn.enaium.jimmer.buddy.utility.AptKt$createAnnotationValue$2.getValue(apt.kt:863)
	at cn.enaium.jimmer.buddy.utility.AptKt$createAnnotationValue$2.toString(apt.kt:884)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableProp.<init>(ImmutableProp.java:302)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableType.<init>(ImmutableType.java:277)
	at org.babyfish.jimmer.apt.Context.getImmutableType(Context.java:273)
	at org.babyfish.jimmer.apt.Context.getImmutableType(Context.java:282)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableProp.getTargetType(ImmutableProp.java:781)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableProp.getIdViewBaseProp(ImmutableProp.java:922)
	at org.babyfish.jimmer.apt.dto.DtoGenerator.converterMetadataOf(DtoGenerator.java:1932)
	at org.babyfish.jimmer.apt.dto.DtoGenerator.getPropTypeName(DtoGenerator.java:1473)
	at org.babyfish.jimmer.apt.dto.DtoGenerator.isSimpleProp(DtoGenerator.java:780)
	at org.babyfish.jimmer.apt.dto.DtoGenerator.addAccessorField(DtoGenerator.java:603)
	at org.babyfish.jimmer.apt.dto.DtoGenerator.addMembers(DtoGenerator.java:246)
	at org.babyfish.jimmer.apt.dto.DtoGenerator.generate(DtoGenerator.java:173)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invokeSuspend$lambda$0(JimmerBuddy.kt:454)
	at com.intellij.openapi.application.ReadAction.lambda$run$1(ReadAction.java:54)
	at com.intellij.openapi.application.impl.AppImplKt$rethrowCheckedExceptions$2.invoke(appImpl.kt:126)
	at com.intellij.platform.locking.impl.NestedLocksThreadingSupport.runReadAction(NestedLocksThreadingSupport.kt:856)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:1069)
	at com.intellij.openapi.application.ReadAction.compute(ReadAction.java:66)
	at com.intellij.openapi.application.ReadAction.run(ReadAction.java:53)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invokeSuspend(JimmerBuddy.kt:429)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invoke(JimmerBuddy.kt)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invoke(JimmerBuddy.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:44)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:166)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.platform.util.progress.ProgressPipeImpl.collectProgressUpdates(ProgressPipe.kt:43)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2$2.invokeSuspend(PlatformTaskSupport.kt:120)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2$2.invoke(PlatformTaskSupport.kt)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2$2.invoke(PlatformTaskSupport.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:44)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:157)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2.invokeSuspend(PlatformTaskSupport.kt:119)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2.invoke(PlatformTaskSupport.kt)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2.invoke(PlatformTaskSupport.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:44)
	at kotlinx.coroutines.CoroutineScopeKt.coroutineScope(CoroutineScope.kt:285)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport.withBackgroundProgressInternal(PlatformTaskSupport.kt:103)
	at com.intellij.platform.ide.progress.TasksKt.withBackgroundProgress(tasks.kt:83)
	at com.intellij.platform.ide.progress.TasksKt.withBackgroundProgress(tasks.kt:23)
	at com.intellij.openapi.progress.TasksKt.withBackgroundProgress(tasks.kt:34)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace.dtoProcessJava(JimmerBuddy.kt:426)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$init$1.invokeSuspend(JimmerBuddy.kt:213)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:610)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runDefaultDispatcherTask(CoroutineScheduler.kt:1194)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:906)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:775)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:762)
Caused by: java.lang.IllegalAccessException: class cn.enaium.jimmer.buddy.utility.AptKt cannot access a member of class cn.enaium.jimmer.buddy.Utility$21_Proxy with modifiers "public"
	at java.base/jdk.internal.reflect.Reflection.newIllegalAccessException(Reflection.java:394)
	at java.base/java.lang.reflect.AccessibleObject.checkAccess(AccessibleObject.java:714)
	at java.base/java.lang.reflect.Method.invoke(Method.java:571)
	at cn.enaium.jimmer.buddy.utility.AptKt.findAnnotation$lambda$0$2(apt.kt:747)
	at cn.enaium.jimmer.buddy.Utility$21_Proxy.value(Utility.java)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	... 54 more

[2026-01-29 16:53:56] DtoProcessJava Project:community-serve:main APTOptions:{}
[2026-01-29 16:53:56] DtoProcessJava Project:community-serve:main APTOptions:{}
[2026-01-29 16:53:56] java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:118)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at cn.enaium.jimmer.buddy.utility.AptKt$createAnnotationMirror$1.getElementValues$lambda$4$0(apt.kt:930)
	at cn.enaium.jimmer.buddy.utility.AptKt$createAnnotationValue$2.getValue(apt.kt:863)
	at cn.enaium.jimmer.buddy.utility.AptKt$createAnnotationValue$2.toString(apt.kt:884)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableProp.<init>(ImmutableProp.java:302)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableType.<init>(ImmutableType.java:277)
	at org.babyfish.jimmer.apt.Context.getImmutableType(Context.java:273)
	at org.babyfish.jimmer.apt.Context.getImmutableType(Context.java:282)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableProp.getTargetType(ImmutableProp.java:781)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableProp.getIdViewBaseProp(ImmutableProp.java:922)
	at org.babyfish.jimmer.apt.dto.DtoGenerator.converterMetadataOf(DtoGenerator.java:1932)
	at org.babyfish.jimmer.apt.dto.DtoGenerator.getPropTypeName(DtoGenerator.java:1473)
	at org.babyfish.jimmer.apt.dto.DtoGenerator.isSimpleProp(DtoGenerator.java:780)
	at org.babyfish.jimmer.apt.dto.DtoGenerator.addAccessorField(DtoGenerator.java:603)
	at org.babyfish.jimmer.apt.dto.DtoGenerator.addMembers(DtoGenerator.java:246)
	at org.babyfish.jimmer.apt.dto.DtoGenerator.generate(DtoGenerator.java:173)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invokeSuspend$lambda$0(JimmerBuddy.kt:454)
	at com.intellij.openapi.application.ReadAction.lambda$run$1(ReadAction.java:54)
	at com.intellij.openapi.application.impl.AppImplKt$rethrowCheckedExceptions$2.invoke(appImpl.kt:126)
	at com.intellij.platform.locking.impl.NestedLocksThreadingSupport.runReadAction(NestedLocksThreadingSupport.kt:856)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:1069)
	at com.intellij.openapi.application.ReadAction.compute(ReadAction.java:66)
	at com.intellij.openapi.application.ReadAction.run(ReadAction.java:53)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invokeSuspend(JimmerBuddy.kt:429)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invoke(JimmerBuddy.kt)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invoke(JimmerBuddy.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:44)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:166)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.platform.util.progress.ProgressPipeImpl.collectProgressUpdates(ProgressPipe.kt:43)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2$2.invokeSuspend(PlatformTaskSupport.kt:120)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2$2.invoke(PlatformTaskSupport.kt)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2$2.invoke(PlatformTaskSupport.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:44)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:157)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2.invokeSuspend(PlatformTaskSupport.kt:119)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2.invoke(PlatformTaskSupport.kt)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2.invoke(PlatformTaskSupport.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:44)
	at kotlinx.coroutines.CoroutineScopeKt.coroutineScope(CoroutineScope.kt:285)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport.withBackgroundProgressInternal(PlatformTaskSupport.kt:103)
	at com.intellij.platform.ide.progress.TasksKt.withBackgroundProgress(tasks.kt:83)
	at com.intellij.platform.ide.progress.TasksKt.withBackgroundProgress(tasks.kt:23)
	at com.intellij.openapi.progress.TasksKt.withBackgroundProgress(tasks.kt:34)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace.dtoProcessJava(JimmerBuddy.kt:426)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$init$1.invokeSuspend(JimmerBuddy.kt:213)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:610)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runDefaultDispatcherTask(CoroutineScheduler.kt:1194)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:906)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:775)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:762)
Caused by: java.lang.IllegalAccessException: class cn.enaium.jimmer.buddy.utility.AptKt cannot access a member of class cn.enaium.jimmer.buddy.Utility$21_Proxy with modifiers "public"
	at java.base/jdk.internal.reflect.Reflection.newIllegalAccessException(Reflection.java:394)
	at java.base/java.lang.reflect.AccessibleObject.checkAccess(AccessibleObject.java:714)
	at java.base/java.lang.reflect.Method.invoke(Method.java:571)
	at cn.enaium.jimmer.buddy.utility.AptKt.findAnnotation$lambda$0$2(apt.kt:747)
	at cn.enaium.jimmer.buddy.Utility$21_Proxy.value(Utility.java)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	... 54 more

[2026-01-29 16:53:56] DtoProcessJava Project:community-serve:main APTOptions:{}
[2026-01-29 16:53:56] java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:118)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at cn.enaium.jimmer.buddy.utility.AptKt$createAnnotationMirror$1.getElementValues$lambda$4$0(apt.kt:930)
	at cn.enaium.jimmer.buddy.utility.AptKt$createAnnotationValue$2.getValue(apt.kt:863)
	at cn.enaium.jimmer.buddy.utility.AptKt$createAnnotationValue$2.toString(apt.kt:884)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableProp.<init>(ImmutableProp.java:302)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableType.<init>(ImmutableType.java:277)
	at org.babyfish.jimmer.apt.Context.getImmutableType(Context.java:273)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invokeSuspend$lambda$0(JimmerBuddy.kt:452)
	at com.intellij.openapi.application.ReadAction.lambda$run$1(ReadAction.java:54)
	at com.intellij.openapi.application.impl.AppImplKt$rethrowCheckedExceptions$2.invoke(appImpl.kt:126)
	at com.intellij.platform.locking.impl.NestedLocksThreadingSupport.runReadAction(NestedLocksThreadingSupport.kt:856)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:1069)
	at com.intellij.openapi.application.ReadAction.compute(ReadAction.java:66)
	at com.intellij.openapi.application.ReadAction.run(ReadAction.java:53)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invokeSuspend(JimmerBuddy.kt:429)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invoke(JimmerBuddy.kt)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invoke(JimmerBuddy.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:44)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:166)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.platform.util.progress.ProgressPipeImpl.collectProgressUpdates(ProgressPipe.kt:43)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2$2.invokeSuspend(PlatformTaskSupport.kt:120)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2$2.invoke(PlatformTaskSupport.kt)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2$2.invoke(PlatformTaskSupport.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:44)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:157)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2.invokeSuspend(PlatformTaskSupport.kt:119)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2.invoke(PlatformTaskSupport.kt)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2.invoke(PlatformTaskSupport.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:44)
	at kotlinx.coroutines.CoroutineScopeKt.coroutineScope(CoroutineScope.kt:285)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport.withBackgroundProgressInternal(PlatformTaskSupport.kt:103)
	at com.intellij.platform.ide.progress.TasksKt.withBackgroundProgress(tasks.kt:83)
	at com.intellij.platform.ide.progress.TasksKt.withBackgroundProgress(tasks.kt:23)
	at com.intellij.openapi.progress.TasksKt.withBackgroundProgress(tasks.kt:34)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace.dtoProcessJava(JimmerBuddy.kt:426)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$init$1.invokeSuspend(JimmerBuddy.kt:213)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:610)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runDefaultDispatcherTask(CoroutineScheduler.kt:1194)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:906)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:775)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:762)
Caused by: java.lang.IllegalAccessException: class cn.enaium.jimmer.buddy.utility.AptKt cannot access a member of class cn.enaium.jimmer.buddy.Utility$21_Proxy with modifiers "public"
	at java.base/jdk.internal.reflect.Reflection.newIllegalAccessException(Reflection.java:394)
	at java.base/java.lang.reflect.AccessibleObject.checkAccess(AccessibleObject.java:714)
	at java.base/java.lang.reflect.Method.invoke(Method.java:571)
	at cn.enaium.jimmer.buddy.utility.AptKt.findAnnotation$lambda$0$2(apt.kt:747)
	at cn.enaium.jimmer.buddy.Utility$21_Proxy.value(Utility.java)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	... 45 more

[2026-01-29 16:53:56] DtoProcessJava Project:community-serve:main APTOptions:{}
[2026-01-29 16:53:57] DtoProcessJava Project:community-serve:main APTOptions:{}
[2026-01-29 16:53:57] java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:118)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at cn.enaium.jimmer.buddy.utility.AptKt$createAnnotationMirror$1.getElementValues$lambda$4$0(apt.kt:930)
	at cn.enaium.jimmer.buddy.utility.AptKt$createAnnotationValue$2.getValue(apt.kt:863)
	at cn.enaium.jimmer.buddy.utility.AptKt$createAnnotationValue$2.toString(apt.kt:884)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableProp.<init>(ImmutableProp.java:302)
	at org.babyfish.jimmer.apt.immutable.meta.ImmutableType.<init>(ImmutableType.java:277)
	at org.babyfish.jimmer.apt.Context.getImmutableType(Context.java:273)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invokeSuspend$lambda$0(JimmerBuddy.kt:452)
	at com.intellij.openapi.application.ReadAction.lambda$run$1(ReadAction.java:54)
	at com.intellij.openapi.application.impl.AppImplKt$rethrowCheckedExceptions$2.invoke(appImpl.kt:126)
	at com.intellij.platform.locking.impl.NestedLocksThreadingSupport.runReadAction(NestedLocksThreadingSupport.kt:856)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:1069)
	at com.intellij.openapi.application.ReadAction.compute(ReadAction.java:66)
	at com.intellij.openapi.application.ReadAction.run(ReadAction.java:53)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invokeSuspend(JimmerBuddy.kt:429)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invoke(JimmerBuddy.kt)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$dtoProcessJava$2.invoke(JimmerBuddy.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:44)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:166)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.platform.util.progress.ProgressPipeImpl.collectProgressUpdates(ProgressPipe.kt:43)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2$2.invokeSuspend(PlatformTaskSupport.kt:120)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2$2.invoke(PlatformTaskSupport.kt)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2$2.invoke(PlatformTaskSupport.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:44)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:157)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2.invokeSuspend(PlatformTaskSupport.kt:119)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2.invoke(PlatformTaskSupport.kt)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport$withBackgroundProgressInternal$2.invoke(PlatformTaskSupport.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:44)
	at kotlinx.coroutines.CoroutineScopeKt.coroutineScope(CoroutineScope.kt:285)
	at com.intellij.openapi.progress.impl.PlatformTaskSupport.withBackgroundProgressInternal(PlatformTaskSupport.kt:103)
	at com.intellij.platform.ide.progress.TasksKt.withBackgroundProgress(tasks.kt:83)
	at com.intellij.platform.ide.progress.TasksKt.withBackgroundProgress(tasks.kt:23)
	at com.intellij.openapi.progress.TasksKt.withBackgroundProgress(tasks.kt:34)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace.dtoProcessJava(JimmerBuddy.kt:426)
	at cn.enaium.jimmer.buddy.JimmerBuddy$Workspace$init$1.invokeSuspend(JimmerBuddy.kt:213)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:610)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runDefaultDispatcherTask(CoroutineScheduler.kt:1194)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:906)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:775)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:762)
Caused by: java.lang.IllegalAccessException: class cn.enaium.jimmer.buddy.utility.AptKt cannot access a member of class cn.enaium.jimmer.buddy.Utility$21_Proxy with modifiers "public"
	at java.base/jdk.internal.reflect.Reflection.newIllegalAccessException(Reflection.java:394)
	at java.base/java.lang.reflect.AccessibleObject.checkAccess(AccessibleObject.java:714)
	at java.base/java.lang.reflect.Method.invoke(Method.java:571)
	at cn.enaium.jimmer.buddy.utility.AptKt.findAnnotation$lambda$0$2(apt.kt:747)
	at cn.enaium.jimmer.buddy.Utility$21_Proxy.value(Utility.java)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	... 45 more



### Intellij Version

2025.3.1.1

### Self-contained Reproducer Project

无，插件下载下来就报错

--- a/core/src/main/kotlin/cn/enaium/jimmer/buddy/utility/apt.kt
+++ b/core/src/main/kotlin/cn/enaium/jimmer/buddy/utility/apt.kt
@@ -22,6 +22,7 @@ import com.intellij.psi.*
 import com.intellij.psi.search.searches.ClassInheritorsSearch
 import com.intellij.psi.util.PsiUtil
 import net.bytebuddy.ByteBuddy
+import net.bytebuddy.description.modifier.Visibility
 import net.bytebuddy.implementation.FixedValue
 import net.bytebuddy.implementation.InvocationHandlerAdapter
 import net.bytebuddy.matcher.ElementMatchers
@@ -50,7 +51,6 @@ import javax.lang.model.util.Elements
 import javax.lang.model.util.Types
 import javax.tools.*
 import javax.tools.JavaFileObject.Kind
-import kotlin.collections.mapNotNull
 import kotlin.io.path.Path
 
 /**
@@ -737,6 +737,7 @@ private fun PsiAnnotation.findAnnotation(): Annotation? = when (qualifiedName) {
 }?.let {
     ByteBuddy()
         .redefine(it.javaClass)
+        .modifiers(Visibility.PUBLIC)
         .name("${it.javaClass.name}_Proxy")
         .method(ElementMatchers.namedOneOf(*it.javaClass.methods.filter { f ->
             Any::class.java.methods.any { it.name == f.name }.not() && f.name != "annotationType"
@@ -764,12 +765,15 @@ private fun PsiAnnotation.findAnnotation(): Annotation? = when (qualifiedName) {
     val classLoader = MyClassLoader()
     val annotationUnloaded = ByteBuddy()
         .makeAnnotation()
+        .modifiers(Visibility.PUBLIC)
         .name(qualifiedName)
         .make()
     map[qualifiedName] = annotationUnloaded.bytes
 
     val proxyName = qualifiedName + "_Proxy"
-    val proxyAnnotationUnloaded = ByteBuddy().subclass(Object::class.java)
+    val proxyAnnotationUnloaded = ByteBuddy()
+        .subclass(Object::class.java)
+        .modifiers(Visibility.PUBLIC)
         .name(proxyName)
         .implement(java.lang.annotation.Annotation::class.java)
         .method(named("annotationType"))
@@ -860,16 +864,20 @@ private fun createAnnotationValue(
 ): AnnotationValue {
     return object : AnnotationValue {
         override fun getValue(): Any? {
-            return value()?.let {
-                if (it is Class<*>) {
-                    if (it == Void::class.java) {
-                        "void"
+            return try {
+                value()?.let {
+                    if (it is Class<*>) {
+                        if (it == Void::class.java) {
+                            "void"
+                        } else {
+                            it.name
+                        }
                     } else {
-                        it.name
+                        it
                     }
-                } else {
-                    it
                 }
+            } catch (_: Throwable) {
+                null
             }
         }
 
@@ -923,18 +931,12 @@ private fun PsiClass.createAnnotationMirror(
                 anno.javaClass.methods.filter { f ->
                     Any::class.java.methods.any { it.name == f.name }.not() && f.name != "annotationType"
                 }.mapNotNull { method ->
-                    try {
-                        createAnnotationValue {
-                            anno.javaClass.getMethod(method.name).also {
-                                it.isAccessible = true
-                            }.invoke(anno)
-                        }
-                    } catch (_: Throwable) {
-                        null
-                    }?.let {
-                        createExecutableElement(
-                            getSimpleName = { createName(method.name) },
-                        ) to it
+                    createExecutableElement(
+                        getSimpleName = { createName(method.name) },
+                    ) to createAnnotationValue {
+                        anno.javaClass.getMethod(method.name).also {
+                            it.isAccessible = true
+                        }.invoke(anno)
                     }
                 }.toMap()
             } else {

--- a/gradle/libs.versions.toml
+++ b/gradle/libs.versions.toml
@@ -1,8 +1,8 @@
 [versions]
 kotlin = "2.3.0"
 ksp = "2.3.0"
-intellij = "2.9.0"
-jimmer = "0.9.119"
+intellij = "2.11.0"
+jimmer = "0.9.120"
 byteBuddy = "1.17.1"
 kotlinpoet = "2.0.0"
 javapoet = "1.13.0"

--- a/gradle/wrapper/gradle-wrapper.properties
+++ b/gradle/wrapper/gradle-wrapper.properties
@@ -1,6 +1,6 @@
 #Thu Feb 13 20:15:24 CST 2025
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-9.2.0-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-9.3.1-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists

--- a/since/253/gradle.properties
+++ b/since/253/gradle.properties
@@ -1,6 +1,6 @@
 supportsK1=false
 supportsK2=true
 platformType=IU
-platformVersion=2025.3.1
+platformVersion=2025.3.2
 sinceBuild=253
 untilBuild=262.*

